staging/src/k8s.io/kube-scheduler/config/v1alpha2/types_pluginargs.go

```diff
@@ -116,6 +125,13 @@ type ResourceSpec struct {
 	Weight int64
 }
 
+// UnmarshalJSON provides case insensitive unmarshalling for the type.
+// TODO remove when copying to v1beta1.
+func (t *ResourceSpec) UnmarshalJSON(data []byte) error {
+	type internal *ResourceSpec
+	return gojson.Unmarshal(data, internal(t))
+}
```

Comment:
is a test needed to demonstrate this works?

Reply:
They are in scheme_test.go